test/dev.md

```diff
@@ -58,7 +58,7 @@ We welcome contributions to kURL. We appreciate your time and help.
 cd kurl
 # validate your expected changes in install.sh|upgrade.sh and|or addons|packages
 # run installation
-sudo ./install.sh
+sudo ./install.sh kurl-install-directory=.
```

Comment:
this is a change to dev env. is there a better way to do this?

Reply:
You could change [this](https://github.com/replicatedhq/kURL/blob/master/bin/watchrsync.js#L28) to `${remote}:/var/lib/kurl`

Reply:
I explored that route. i think this is preferable because if i change that i have to add a bunch of sudos to those tar commands to download packages